app/src/main/java/com/stevesoltys/seedvault/metadata/MetadataManager.kt

```diff
@@ -312,7 +312,7 @@ internal class MetadataManager(
     private fun getUserName(): String? {
         val perm = "android.permission.QUERY_USERS"
         return if (context.checkSelfPermission(perm) == PERMISSION_GRANTED) {
-            val userManager = context.getSystemService(UserManager::class.java)
+            val userManager = context.getSystemService(UserManager::class.java) ?: return false
```

Comment:
Am I misunderstanding this, or is this trying to return `false` as a `String?`?

Reply:
that was supposed to be `null`. sorry. I did this right from the web ui as I am working on another branch at the moment.